### DISCUSSION TITLE
Bulk convert evry PNG in the folder to JPG

### DISCUSSION THREAD

**Discussion by AsterYawn** [ImageMagick/ImageMagick#6757] [Help] [answered]
### ImageMagick version

7.1.1

### Operating system, version and so on

Windows 11

### Description

Need help with writing a command that will convert all PNG files in a folder to JPG, replacing them or putting them in a new folder. The images need to keep their quality more or less. That's it, thank you!

**Comment by snibgo** [comment, marked as answer]
In the directory that contains your PNG files:
```
magick mogrify -format jpg -quality 100 *.png
```
This will create one JPG from each PNG, in the same directory. It will not remove the PNG files. You can use a lower quality number for smaller but lower quality JPGs. You can also "`-path mydir`" to write the JPG to whatever directory you want.

**Reply by AsterYawn** [reply]
Sorry, I'm a bit new to command prompts, how do I run this on a specific folder then? 
![image](https://github.com/ImageMagick/ImageMagick/assets/147052539/6297076b-a7f9-4fc0-a1c7-fddf160c7026)

**Comment by snibgo** [comment]
I suggest you `cd` to the directory that contains your PNG files. `cd` is like moving to a different room in your house. If you want to find a book, it is easier to do that when you are in the room.

Type "`help cd`" for help on `cd`, or just "`help`" to list many available commands.

An alternative is to specify the directory that contains your PNG files like this:
```
magick mogrify -format jpg -quality 100 C:\Users\Alan\mypictures\*.png
```

**Reply by AsterYawn** [reply]
got it, works now, thank you!
